### ISSUE TITLE
Bug statisticsDomain

### ISSUE THREAD

**Issue by cristinamanzano** [dCache/dcache#6603]
When trying to start dCache the following output is logged in the statisticsDomain log:

```
Apr 22 13:44:56 zam288 dcache@statisticsDomain: 22 Apr 2022 13:44:56 (System) [] Bug detected in dCache; please report this to <support@dcache.org> with the following information.
Apr 22 13:44:56 zam288 dcache@statisticsDomain: java.lang.IllegalArgumentException: Either <baseDirectory> or <htmlBase> doesn't exist
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at diskCacheV111.services.PoolStatisticsV0.starting(PoolStatisticsV0.java:231)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at dmg.cells.nucleus.CellAdapter.prepareStartup(CellAdapter.java:681)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at dmg.cells.nucleus.CellNucleus.doStart(CellNucleus.java:899)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at dmg.cells.nucleus.CellNucleus.lambda$wrapLoggingContext$3(CellNucleus.java:735)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at org.dcache.util.BoundedExecutor$Worker.run(BoundedExecutor.java:247)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at dmg.cells.nucleus.CellNucleus.lambda$wrapLoggingContext$2(CellNucleus.java:727)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: at java.base/java.lang.Thread.run(Thread.java:829)
Apr 22 13:44:56 zam288 dcache@statisticsDomain: 22 Apr 2022 13:44:56 (System) [] Failure at startup: (666) URL [file:/usr/share/dcache/services/statistics.batch]: line 25: (1) Command failed: java.lang.IllegalArgumentException: Either <baseDirectory> or <htmlBase> doesn't exist
Apr 22 13:44:56 zam288 dcache@statisticsDomain: WARNING: An illegal reflective access operation has occurred
Apr 22 13:44:56 zam288 dcache@statisticsDomain: WARNING: Illegal reflective access by org.nustaq.serialization.FSTClazzInfo (file:/usr/share/dcache/classes/fst-2.56.jar) to field java.lang.String.value
Apr 22 13:44:56 zam288 dcache@statisticsDomain: WARNING: Please consider reporting this to the maintainers of org.nustaq.serialization.FSTClazzInfo
Apr 22 13:44:56 zam288 dcache@statisticsDomain: WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
Apr 22 13:44:56 zam288 dcache@statisticsDomain: WARNING: All illegal access operations will be denied in a future release
```

Some relevant information: we are running dCache 7.2.15, OpenJDK 11, CentOS 7.9

**Comment by lemora** [comment]
Hi.
Were there any other messages logged before the stack trace?
Which are the values you have configured for `statistics.location`, alternatively `dcache.paths.statistics`? Are there any files in these locations?

**Comment by cristinamanzano** [comment]
Hi, apologies, I made a beginner's mistake: I was somehow sure the path for the statistics existed but actually I had forgotten to create it. So I created `/data/dcache/statistics` and changed the ownership to the `dcache` user (althought I am not sure this last is needed) and this seemed to solve the problem. The statisticsDomain is running now and the stack trace is not logged anymore. Thanks for the hint.
